<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordDialog</class>
 <widget class="QDialog" name="WordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="wordLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="meanLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>78</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="trueButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>认识</string>
   </property>
  </widget>
  <widget class="QPushButton" name="falseButton">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>190</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
